src/harness/fake_sas.py

```diff
@@ -342,16 +357,19 @@ def do_GET(self):
     self.wfile.write(json.dumps(response))
 
 
-def RunFakeServer(version, is_ecc):
+def RunFakeServer(version, is_ecc, ca_cert):
```

Comment:
this change is not needed for this PR, so better to revert this file

Reply:
Changes   in this file are not relevant for this PR, we can remove the file from Pull   Request

Reply:
this change has been removed